packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -380,6 +385,20 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
   }
 
+  /// Sets the description while the camera is recording
+  Future<void> setDescriptionWhileRecording(
+      CameraDescription description) async {
+    if (!value.isRecordingVideo) {
+      throw CameraException(
+        'Video was not being recording',
```

Comment:
I think this was supposed to be `Video was not being recorded`

Reply:
than u sir for ur feedback but it doesnt work. when i do ur trick. maybe i made something wrong can you explain me clearly if you avaiable pleas it make me very happy. I am working on it for a week but i dost' work.